specs/ChainSpec/Upgradability.md

```diff
@@ -0,0 +1,71 @@
+# Upgradability
+
+This part of specification describes specifics of upgrading the protocol, and touches on few different parts of the system.
+
+High level upgradability consists of three parts:
```

Comment:
I feel you mean "three different levels" of upgrades, a higher level means touches more parts of system". so this sentence is better to be "Upgradability consists of three levels:"